doc/classes/TextEdit.xml

```diff
@@ -259,6 +268,7 @@
 			<argument index="0" name="option" type="int">
 			</argument>
 			<description>
+				Triggers a right click menu action by the specified index. See [MENU_*] enums for a list of available indexes.
```

Comment:
`[MENU_*]` will not work. It should be "See the [enum MenuItems] enum values for...".